website/pages/blog/optimize-bundle-size-with-swc-and-graphql-codegen.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Optimize your Bundle Size with SWC and GraphQL Codegen
```

Comment:
does it also give performance benefits?  if so, I think we should make sure to highlight it in the blog post and also in the title

Reply:
In the end websites can load faster, because the bundle size is smaller. I'm not sure if I would call it more performant, as I would associate it with faster working code. 

Maybe we can highlight faster page loading times, like this: _Optimize your Bundle Size with SWC and GraphQL Codegen for Faster Page Loading_.

In my head I would already associate faster page loading times if I see the original title, so I'm not sure if we need to highlight it more

Reply:
I meant in terms of dev performance, like build times and stuff like that?

Reply:
I think dev performance is even a little bit less (but should be unnoticeable) because of an extra compilation step if you would compare it to not using a SWC or Babel plugin. But it is definitely more performant if you were previously using Babel. So I think we could leave it as is :)